### DISCUSSION TITLE
Key points of next gov security and voting power.

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1284] [Ideas]
1. RING working as the gov token, can be used to generate voting power with voluntary-locking(conviction by time)
2. Voting power should be locked to some account and should not be trade (Not a token).
3. Use (precompile) contract to lock the RING and providing voting power calculating and interface.
  - extend(ring_amount, locking_period), to make it simple, only one item for each account, extend the voting power with larger ring_amount or locking_period, or cancel if no voting going on.
  - The inteface can be used to integrate more composite voting sources (collating, deposit? looks not good though.)
  - Abandon using substrate balance lock, because substrate balance lock is confuse to evm users.
5. Voting power can be used to vote council, referenda, gov v2 etc at the same time.
6. Voting power is to guard the gov security, should be incentive just like collator staking (? Still questionable, resulting more proposals, and risks of governance communication )
7. Delegate.

Ref:
1. https://cryptopotato.com/what-are-the-risks-of-governance-tokens-opinion/?amp
2. https://wiki.polkadot.network/docs/learn-polkadot-opengov#voluntary-locking
3. https://www.quora.com/Can-you-stake-the-uni-token-to-earn-rewards-and-also-use-them-for-governance-at-the-same-time

**Comment by hackfisher** [comment]
What if voting power is incentives but do not do voting, this will still not protect the gov participant?

**Comment by hackfisher** [comment]
https://en.wikipedia.org/wiki/Quadratic_voting#:~:text=Quadratic%20voting%20is%20a%20collective,Condorcet%20paradox%20and%20majority%20rule.

**Comment by hackfisher** [comment]
Updates after review meeting:

1. Delegate is important for increasing voting participant and professional contribution, and delegate must be set before locking for voting power.
2. Only voting_power with delegation should have incentive, further voting action on proposals should not have incentive as it could distort the behaviour of the voting process by adding meaningless noise or interfere with the normal game process of the proposal
3. delegatee can carry a fixed commission(5%) from its supporters, to incentive he to behaviour responsible, professional and actively. (But this could have side effective of supporters choose themself as delegatee, so there would be better some other incentives from proposals for delegate to participating regardless of voting yes or no, in that case, only those professional ones who active can catch this part, and will make those ones who randomly select their delegatee consider carefully)
(Incentives happening offchain from proposals some times can be seen as bribing or not?)
(Some members of collective like council may able to initiate incentive voters of some important proposals, incentive may come from system commission 1%)

**Comment by 2xJDubs** [comment]
Staked RING needs to be accessible for voting purposes.  Right now some of our most active community members are not active in governance because they have very little free RING.  This needs to be considered soon as the longer we go without engaging the community in voting the harder it will be to bring them back to governance.

**Comment by Laki389** [comment]
I agree with @2xJDubs, staked and locked Ring must be able to participate in voting and decision making in Darwinia 2.0 as soon as possible.

As it stands now, the so-called forced white-hat hacker from the Crab network (who also got Ring) has more voting power than community members who have locked Ring for 3 years.
It doesn't make sense to me.

As Darwinia 2.0 slowly moves towards open gov, it should be possible for everyone to vote and participate in decision-making, so the voted proposals will have more weight and legitimacy, and at the same time it will give a feeling to the members of the community that they contribute and participate in decision-making, and will also become more active and aware of the course of events in Darwinia.

**Comment by SasoLithops** [comment]
Much agree with Jesse but I understand now there is sadly nothing we can do with GOV1 governance model we have. But I would suggest that only locked Ring (when we have OPENGOV) be the one you can vote with.

**Comment by hackfisher** [comment]
Ref: https://github.com/darwinia-network/darwinia/issues/1070
